compiler/src/codegen/compcore.ml

```diff
@@ -129,6 +129,9 @@ let wrap_int64 n = add_dummy_loc (Values.I64Value.to_value n)
 let wrap_float32 n = add_dummy_loc (Values.F32Value.to_value n)
 let wrap_float64 n = add_dummy_loc (Values.F64Value.to_value n)
 
+let grain_number_max = 0x3fffffff
+let grain_number_min = -0x3fffffff
```

Comment:
Is it possible to write this without the negative? Or at least a comment with the full hex of the negative

Reply:
The native numbers in OCaml extend past 32 bits, so we can't write out the literal just here, but we could do `Int32.to_int 0xC0000001l` if you'd like that better?

Reply:
Hmm. Maybe just putting `(* 0xC0000001 *)` next to it would be sufficient. I mostly just want it so that we can quickly identify what the memory layout of the number is, so it's not important that the actual source has the proper constant.